libcontainer/rootfs_linux.go

```diff
@@ -594,6 +595,37 @@ func createDevices(config *configs.Config) error {
 }
 
 func bindMountDeviceNode(dest string, node *configs.Device) error {
+	var st unix.Stat_t
+	if err := unix.Lstat(dest, &st); err != nil {
+		if !os.IsNotExist(err) {
+			return errors.Wrapf(err, "could not stat existing path for device node: %q", dest)
+		}
+	} else if os.FileMode(st.Mode)&os.ModeSymlink == os.ModeSymlink {
```

Comment:
```suggestion
	} else if os.FileMode(st.Mode) & os.ModeSymlink == os.ModeSymlink {
```

Reply:
We require all code to follow `gofmt`, which your suggestion doesn't (yeah, I also think it looks a bit silly but that's what the Go dev's thought the Go style should look like).